src/backend/api/reservations.js

```diff
@@ -0,0 +1,93 @@
+const express = require("express");
+const router = express.Router();
+const knex = require("../database");
+
+router.get("/", async (request, response) => {
+  try {
+    // knex syntax for selecting things. Look up the documentation for knex for further info
+    if (request.query.hasOwnProperty('id')) {
+      const requestId = Number(request.query.id);
+      const reservations = await knex("reservations")
```

Comment:
In all the `id` querying instances, I would add a check if the `requestId` turned out to be actually a number.